### PR TITLE
Add config manifest for serial e2e test.

### PR DESCRIPTION
**Pull request by nilo19**
Add config manifest for serial e2e test.

**Release note**:
```
None
```

### CODE REVIEW

File: tests/k8s-azure/manifest/linux-vmss-serial.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        "masterProfile": {
+            "count": 1,
+            "dnsPrefix": "{dnsPrefix}",
+            "vmSize": "Standard_F2"
+        },
+        "agentPoolProfiles": [
+            {
+                "name": "agentpool1",
+                "count": 2,
+                "vmSize": "Standard_F2",
```
Comment (feiskyer) [inline tests/k8s-azure/manifest/linux-vmss-serial.json]: let's change the size here same as the test e2e tests, `Standard_D4s_v3`

File: tests/k8s-azure/manifest/linux-vmss-serial.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                },
+                "apiServerConfig": {
+                    "--enable-admission-plugins": "NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,ResourceQuota,AlwaysPullImages"
+                }
+            }
+        },
+        "masterProfile": {
+            "count": 1,
+            "dnsPrefix": "{dnsPrefix}",
+            "vmSize": "Standard_F2"
```
Comment (feiskyer) [inline tests/k8s-azure/manifest/linux-vmss-serial.json]: `Standard_D4s_v3`

File: tests/k8s-azure/manifest/linux-vmss-serial.json (hunk 1 above)
Comment (nilo19) [reply tests/k8s-azure/manifest/linux-vmss-serial.json]: It'll be overwritten in cloud-provider-azure-config.yaml in test-infra.

Comment (feiskyer) [reply tests/k8s-azure/manifest/linux-vmss-serial.json]: right, but changing here would make it more clear for reading

**Pull request merged**